src/odemis/driver/tmcm.py

```diff
@@ -2272,7 +2374,9 @@ def _parseMessage(self, msg):
             if not 0 <= typ <= 7:
                 self._sendReply(inst, status=3)  # wrong type
                 return
-            # Nothing actually to do
+            # Change internal do value
+            if mot == 2 and 0 <= typ <= 7:
```

Comment:
As the line just above has already checked for `typ`, you can simplify this `if` condition.